Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -368,6 +367,13 @@ ProcessCode TrackFindingAlgorithm::execute(const AlgorithmContext& ctx) const {
   tracksTemp.addColumn<unsigned int>("trackGroup");
   Acts::ProxyAccessor<unsigned int> seedNumber("trackGroup");
 
+  // alternatively you can use `CKF::prepareContainer` but that is not available
+  // here
+  tracks.addColumn<Acts::CombinatorialKalmanFilterTipState>(
+      Acts::s_combinatorialKalmanFilterTipStateColumn);
+  tracksTemp.addColumn<Acts::CombinatorialKalmanFilterTipState>(
+      Acts::s_combinatorialKalmanFilterTipStateColumn);
```

Comment:
Wouldn't this work?

```suggestion
    Acts::CombinatorialKalmanFilter::prepareContainer(tracks);
    Acts::CombinatorialKalmanFilter::prepareContainer(tracksTemp);
```
Maybe it would be clearer to put it before `addColumn<unsigned int>("trackGroup")`. I think it's OK to have separate examples' `addColumn`.

Reply:
I fear we would need the template params of the `Acts::CombinatorialKalmanFilter`. Alternatively we could make the utility a free function I guess?

Reply:
ah yes, I see the problem. That will be a problem for all the users of this function, so it's good to fix it now :) . Can we move it to a free function, eg. `Acts::CombinatorialKalmanFilterPrepareContainer()`.

This is a similar issue with had with the `BranchStopperResult`. Maybe we should have made a CKF namespace, but it's probably too late now.

Reply:
added a fee function as a helper